sitemedia/js/controllers/ittpanel_controller.js

```diff
@@ -14,6 +14,10 @@ export default class extends Controller {
     connect() {
         document.addEventListener("tahqiq-alert", this.boundAlertHandler);
         document.addEventListener("tahqiq-cancel", this.boundCancelHandler);
+        if (this.isDesktop() && this.transcriptionAndTranslationOpen()) {
+            // if transcription + translation both open, align their contents line-by-line
+            this.alignLines();
```

Comment:
👍 

Do you currently support listening for window resize and aligning ( and un-aligning?) when the page crosses the desktop/mobile boundary? Or could that be on the list somewhere?

Reply:
The alignment styles are not applied in mobile, but the code doesn't handle resizing within desktop mode, so added an issue for this! #1431